source/compatibility.tex

```diff
@@ -2547,7 +2548,7 @@
 \indextext{bit-field!implementation-defined sign of}%
 Bit-fields of type plain \tcode{int} are signed.
 \rationale
-Leaving the choice of signedness to implementations could lead to
+Leaving the choice of signedness to implementations can lead to
```

Comment:
Similarly, the leaving can't do anything since it doesn't happen.  Maybe

> The signedness needs to be consistent among template specializations.

Reply:
Ditto.